nifi-nar-bundles/nifi-py4j-bundle/nifi-python-extension-api/src/main/python/src/nifiapi/properties.py

```diff
@@ -133,8 +133,9 @@ def __init__(self, name, description, required=False, sensitive=False,
         :param expression_language_scope: documents the scope in which Expression Language is valid. This value must be specified as one of the enum values
                                           in `nifiapi.properties.ExpressionLanguageScope`. A value of `NONE` indicates that Expression Language will not be evaluated for this property.
                                           This is the default. A value of `FLOWFILE_ATTRIBUTES` indicates that FlowFile attributes may be referenced when configuring the property value.
-                                          A value of `LIMITED` indicates that Expression Language may be used but may not reference FlowFile attributes. For example, a value of `${now()}` might
-                                          be used to reference the current date and time, or `${hostname(true)}` might be used to specify the hostname.
+                                          A value of `ENVIRONMENT` indicates that Expression Language may be used and may reference environment variables but may not reference FlowFile attributes.
```

Comment:
ENVIRONMENT in the nifi-api implies "Environment (System properties and Operating System environment variables)". Let's mention both.

Reply:
Yeah, that's fair. If you're good with everything otherwise, is probably easiest to just tweak the text while merging. Otherwise, I can update the PR when I get a chance.

Reply:
already done.  just validating on my local setup the findings are resolved.